### ISSUE TITLE
Feature request - CRN values 

### ISSUE THREAD

**Issue by MohamedRadwan** [tobiasschlagenhaufer/cuScheduling#9]
Could you please add CRN values as part of the table so that it would be easier to select on Carleton Central ?

**Comment by tobiasschlagenhaufer** [comment]
I've debated adding CRN values, do you mean so that you can add the courses easily once you have a schedule you like?

**Comment by MohamedRadwan** [comment]
Yeah so the CRN value would make it much easier to create the schedule on central. Since you can simply copy past the crns and create the schedule instead of having to search for every course.
![image](https://user-images.githubusercontent.com/21133129/85952527-e6846e80-b937-11ea-881f-e34db0fa0772.png)

**Comment by tobiasschlagenhaufer** [comment]
That would be nice for sure, but I'd need to think of a good way of displaying them since there isn't much space on the class itself
